src/components/graphql-provider/__mocks__/graphql-provider.tsx

```diff
@@ -382,11 +542,29 @@ export const GraphQlProvider = ({
   return (
     <MockedProvider
       mocks={[
+        MOCK_PROPOSALS,
+        MOCK_PROPOSALS,
+        MOCK_PROPOSALS,
+        MOCK_PROPOSALS,
+        MOCK_PROPOSALS,
+        MOCK_PROPOSALS,
+        MOCK_PROPOSALS,
+        MOCK_PROPOSALS,
+        ...MOCK_PROPOSAL,
+        ...MOCK_PROPOSAL,
+        ...MOCK_PROPOSAL,
+        ...MOCK_PROPOSAL,
+        ...MOCK_PROPOSAL,
+        ...MOCK_PROPOSAL,
+        ...MOCK_PROPOSAL,
+        ...MOCK_PROPOSAL,
+        ...MOCK_PROPOSAL,
+        ...MOCK_PROPOSAL,
+        ...MOCK_PROPOSAL,
+        ...MOCK_PROPOSAL,
```

Comment:
Need a better solution than this for serving multiple mocks, I said I'd have a look into it before xmas but didn't have the time. Will look now 👍

Reply:
Open issue about this:
https://github.com/apollographql/apollo-feature-requests/issues/274

Potential work around:
https://gist.github.com/geraint-37d5/626f5c68d6f943a9c9d4508c4e18e564

Let's all have a discussion about it in our next standup

Reply:
can this get merged in like this and handled on a seperate PR ?

Reply:
#1117